src/plot_api/subroutines.js

```diff
@@ -43,7 +43,9 @@ function overlappingDomain(xDomain, yDomain, domains) {
 exports.lsInner = function(gd) {
     var fullLayout = gd._fullLayout;
     var gs = fullLayout._size;
+    var pad = gs.p;
     var axList = Plotly.Axes.list(gd);
+    var hasSVGCartesian = fullLayout._has('cartesian');
```

Comment:
To note, this is needed as gl2d now uses cartesian (SVG layers) subplots to handle selections.